packages/web-components/fast-components/src/styles/recipes.ts

```diff
@@ -31,306 +31,489 @@ import {
 import { accentBaseColor } from "../fast-design-system";
 import { FASTDesignSystemProvider } from "../design-system-provider/index";
 
+/**
+ * Behavior to resolve and makes available the neutral-foreground-rest CSS Custom Property.
```

Comment:
nit: "make" not "makes" (turtles all the way down on this file), also should we capitalize the first letter in "Custom Property" or use lowercase?